### ISSUE TITLE
when using MAP, results are not scaled with the average beta_0

### ISSUE THREAD

**Issue by jcohenadad** [neuropoly/spinalcordtoolbox#348]
currently in the MAP method, the weighting of the prior parameter (beta_0) is biased towards the mean value in the white matter.  The smaller the value, the more weighting is given to the prior.

**Comment by jcohenadad** [comment]
I did some investigations by multiply by factors of 0.1, 10, etc. and I confirm that results are not scaled as they should. When using ML, they are scaled.

In MAP, the "Estimated beta per cluster:" is appropriately scaled. Example:
~~~~
x0.1: [ 4.02643333  3.49061413  2.00315126]
x1: [ 40.26433334  34.90614135  20.0315126 ]
x10: [ 402.64333339  349.06141346  200.31512597]
~~~~

**Comment by jcohenadad** [comment]
fixed in d361bd0906f35f23e33a8c5b5ab974f601333854

variance across labels was estimated as a percentage of beta_0, which causes the scaling problem. Now, variance is fixed.
